src/data/setoid/partition.lean

```diff
@@ -28,6 +33,8 @@ Could the design of `finpartition` inform the one of `setoid.is_partition`? Mayb
 changing it from `set (set α)` to `set α` where `[lattice α] [order_bot α]` would make it more
 usable.
 
+Are `finpartition.of_partition` and `is_partition.of_finpartition` enough ?
+
```

Comment:
```suggestion
```
This type of question is best left for the PR description (below the `---`), not doc-comments. I think the second half of the TODO still applies, but you've definitely wrapped up the first half.